plutus-conformance/agda/Spec.hs

```diff
@@ -43,7 +43,11 @@ failingTests = [
     "test-cases/uplc/evaluation/builtin/mkNilPairData"
     , "test-cases/uplc/evaluation/builtin/mkNilData"
     -- this is because agda has the BuiltinVersion=V1; haskell defaults to latest BuilinVersion
-    , "test-cases/uplc/evaluation/builtin/consByteString"
+    , "test-cases/uplc/evaluation/builtin/consByteString/consByteString1"
+    , "test-cases/uplc/evaluation/builtin/consByteString/consByteString2"
+    -- This is because the V2 version of consByteString requires the first
```

Comment:
This is a bit confusing, normally we put comments _above_ the thing they're commenting on?

Reply:
Oh yeah, that's messed up.  I'll fix that.